### ISSUE TITLE
Pop up an incorrect browser window automatically when debugging a Reactsite project on Windows OS

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-azurestaticwebapps#570]
**OS:** Linux/Windows
**Build Version:** 20211105.14
**Regression:** Not a regression

**Repro Steps:**

1. Open a Reactsite project in VS Code.
2. Type `cd app` in TERMINAL window -> Type `npm install`.
3. Navigate to "Run and Debug" view -> Select "Azure Static Web Apps…" in the dropdown list -> Select "SWA: Run app" 
-> Start Debugging.
4. Check whether pop up a correct browser window automatically.

**Expect:**
Pop up a correct browser window automatically.

**Actual:**
Pop up an incorrect browser window automatically.
![image](https://user-images.githubusercontent.com/57662131/140912683-f696ad24-1255-4eeb-a1fe-1a62f355a03e.png)

**More Info:**
On Linux, the browser window doesn't pop up automatically, need to type the address in browser manually.

**Comment by alexweininger** [comment]
Thanks for the issue report. 

Unfortunately, the Chrome window needed for debugging with VS Code must be launched separate from any currently running Chrome window. This would definitely be nice, but currently it isn't possible.

**Comment by nturinski** [comment]
> More Info:
On Linux, the browser window doesn't pop up automatically, need to type the address in browser manually.

What about this?

**Comment by alexweininger** [comment]
@v-xinda Could you try the test on Linux again using this repo? https://github.com/alexweininger/ReactSite
Thanks!

**Comment by v-xinda** [comment]
@nturinski The browser window doesn't pop up automatically when debugging the project on Linux. The above browser window opens after typing the address "http://localhost:4280" in the Chrome browser.

**Comment by v-xinda** [comment]
@alexweininger This issue also reproduces on Linux when debugging the ReacSite project which cloned from https://github.com/alexweininger/ReactSite.

**Comment by v-xinda** [comment]
**OS:** All
For vue-baisc-api / angular-basic-api repo (Clone https://github.com/alexweininger/vue-basic-api / https://github.com/alexweininger/angular-basic-api):
The browser window doesn't pop up automatically when debugging it. Display `hello world` in the page after typing the address in browser manually.
![image](https://user-images.githubusercontent.com/57662131/141103886-e0745294-dc79-4495-a89e-bfad9441030e.png)
For nextjs-basic-api repo (Clone https://github.com/alexweininger/nextjs-basic-api):
The browser window doesn't pop up automatically when debugging it. Display `What Can I Deploy to Static Apps?` in the page after typing the address in browser manually.
![image](https://user-images.githubusercontent.com/57662131/141103961-b0ef4390-c8ae-4da2-8a99-8ba1226510fe.png)
For svelte-basic-api repo (Clone https://github.com/alexweininger/svelte-basic-api):
The browser window doesn't pop up automatically when debugging it. The page is blank after typing the address in browser manually.
![image](https://user-images.githubusercontent.com/57662131/141103765-af544e27-4a11-46e9-95fb-ef8c66c5c69d.png)

**Comment by alexweininger** [comment]
@v-xinda When you run `npx swa --version`, what is the output? And also `func --version`?

And could you try running `npm start` in the root of this workspace with a cloned staticwebdev/angular-basic repo and report the output and behavior?

**Comment by alexweininger** [comment]
> @alexweininger This issue also reproduces on Linux when debugging the ReacSite project which cloned from [alexweininger/ReactSite](https://github.com/alexweininger/ReactSite).

Thanks for the information. This now leads me to believe that there's an issue with the `pwa-chrome` debug type launching Chrome on Linux. But I have yet to be able to reproduce this on my own.

**Comment by v-xinda** [comment]
@alexweininger  
**func version:** 4.0.3928
**SWA CLI version:** 0.8.0
The result of running `npm start` as below:
```
PS C:\Users\AzureTest\Desktop\angular-basic> npm start

> angular-basic@0.0.0 start C:\Users\AzureTest\Desktop\angular-basic
> ng serve

Compiling @angular/core : es2015 as esm2015
Compiling @angular/common : es2015 as esm2015
Compiling @angular/platform-browser : es2015 as esm2015
Compiling @angular/platform-browser-dynamic : es2015 as esm2015
√ Browser application bundle generation complete.

Initial Chunk Files   | Names         |      Size
vendor.js             | vendor        |   2.43 MB
polyfills.js          | polyfills     | 485.71 kB
styles.css, styles.js | styles        | 346.78 kB
main.js               | main          |   7.51 kB
runtime.js            | runtime       |   6.15 kB

                      | Initial Total |   3.26 MB

Build at: 2021-11-11T03:00:47.224Z - Hash: 9adaff48e91098578d2d - Time: 55055ms

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **


√ Compiled successfully.
√ Browser application bundle generation complete.

Initial Chunk Files   | Names  |      Size
styles.css, styles.js | styles | 346.78 kB

4 unchanged chunks

Build at: 2021-11-11T03:00:52.906Z - Hash: 0bafedaf87216d457228 - Time: 3919ms

√ Compiled successfully.
```

**Comment by alexweininger** [comment]
@v-xinda We've added a fix for #574 that we think might also fix this issue. Could you retry this on the most recent build?

Build: [20211110.10](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=40742&view=results)

Thanks! 🤠

**Comment by v-xinda** [comment]
Verified this issue on the build 20211110.10, it doesn't reproduce on Windows and Mac. But it still reproduces on Linux. Serene gives you a Linux machine which reproduced this issue
